azavea/pwd-waterworks-revealed
# Source intro image locally

- Devices without a data or wi-fi connection couldn't access
  the background image used in the intro because it the CSS
  was sourcing it from azavea.com. This changes adds the image
  to the repo and updates the CSS to source the image locally.

**To test:**
- The branch was built into an APK and installed on the nexus tablet. Check it out on there or build it locally.

Connects to #87
